### PR TITLE
chore(build): fix video player in canary site

### PR DESCRIPTION
**Pull request by asudoh**
### Description

An attempt to fix video player in Canary site.

### Changelog

**Changed**

- Usage of `.env` file to make Kaltura keys in effect in our Canary site.

### CODE REVIEW

File: .github/workflows/deploy-web-components-to-canary.yml [hunk 1]
```diff
@@ -20,11 +20,18 @@ jobs:
...
       - name: Build project
         run: yarn build
-      - name: Building @carbon/ibmdotcom-web-components storybook
+      - name: Create .env file
         env:
-          CORS_PROXY: ${{ secrets.CORS_PROXY }}
           KALTURA_PARTNER_ID: ${{ secrets.KALTURA_PARTNER_ID }}
           KALTURA_UICONF_ID: ${{ secrets.KALTURA_UICONF_ID }}
+        run:
+          touch .env
```
Comment (jeffchew) [inline .github/workflows/deploy-web-components-to-canary.yml:28]: If this works, we would have to make adjustments to the other yml file and jobs I created right?

Comment (asudoh) [reply .github/workflows/deploy-web-components-to-canary.yml:28]: Yes, I think so.

File: .github/workflows/deploy-staging.yml [hunk 2]
```diff
@@ -51,11 +51,15 @@ jobs:
         uses: ./.github/actions/install-ibmcloud-cli
       - name: Build project
         run: yarn build
-      - name: Building @carbon/ibmdotcom-web-components (staging) storybook
+      - name: Set env vars
+        uses: ./.github/actions/set-dotenv
+        with:
+          env-file: packages/web-components/.env
```
Comment (jeffchew) [inline .github/workflows/deploy-staging.yml:57]: @asudoh can you make the changes for the other environments like React too? I'm noticing the staging environments are not loading things like the CORS_PROXY, etc.

Comment (asudoh) [reply .github/workflows/deploy-staging.yml:61]: @jeffchew Sure, done!

**Pull request merged**